doc/source/whatsnew/v0.23.0.txt

```diff
@@ -410,6 +410,7 @@ I/O
 - Bug in :func:`read_sas` where a file with 0 variables gave an ``AttributeError`` incorrectly. Now it gives an ``EmptyDataError`` (:issue:`18184`)
 - Bug in :func:`DataFrame.to_latex()` where pairs of braces meant to serve as invisible placeholders were escaped (:issue:`18667`)
 - Bug in :func:`read_json` where large numeric values were causing an ``OverflowError`` (:issue:`18842`)
+- Bug in :func:`DataFrame.to_parquet` where an attempt to write to S3 path fails with ``ValueError`` (:issue:`19134`)
```

Comment:
isn't the user facing issue, attempting to write to a non-existant path? if so say that

Reply:
No, that was my initial mis categorization of the bug. After I've got familiar with the code I actually realized it's both for a new path as well as for existing keys. The failure is when making a write attempt to s3 using S3FileSystem being initialized in read mode.

Sorry for the initial confusion :)

Reply:
ok then say that. if you are a user reading this then a ``ValueError`` is compeltely opaque